### ISSUE TITLE
missing namespace changes in raven upgrade guide

### ISSUE THREAD

**Issue by SimonCropp** [Particular/docs.particular.net#1474]
http://docs.particular.net/nservicebus/ravendb/upgrades/3to4#namespace-change

for example this commit https://github.com/Particular/docs.particular.net/commit/9555a7ae69d4b0648468c0944a4ae8f532ac9bf5

**Comment by SimonCropp** [comment]
@Particular/ravendb-persistence-maintainers can someone look at this

**Comment by SimonCropp** [comment]
not sure what other namespaces have changed.

**Comment by DavidBoike** [comment]
Added to the PoA for Particular/PlatformDevelopment#733

**Comment by MarcinHoppe** [comment]
I examined all the changes and it seems that the `ConnectionParameters` class was moved to `NServiceBus.Persistence.RavenDB`. I think it would be more consistent if it were moved to `NServiceBus`. After all this is something a business developer would use and not an extension point.

Other than that the namespaces look OK and we don't need to update the upgrade guide.

Feel free to merge this [PR](https://github.com/Particular/NServiceBus.RavenDB/pull/230) in case you happen to agree.
